docs/source/user/api/apollon-editor.d.ts

```diff
@@ -1,10 +1,9 @@
 import 'pepjs';
-import { DeepPartial } from 'redux';
 import { Styles } from './components/theme/styles';
 import { Patch } from './services/patcher';
 import { Locale } from './services/editor/editor-types';
 import * as Apollon from './typings';
-import { UMLDiagramType, UMLModel } from './typings';
+import { DeepPartial, UMLDiagramType, UMLModel } from './typings';
```

Comment:
this is a recurring change all over, may I ask why do we need to customise the `DeepPartial` type helper offered by redux? how was it changed?

Reply:
The DeepPartial Type is no longer exported by Redux version 5 but all diagram element constructors are initialized with "DeepPartials" of the diagram element models - therefore, I introduced a self defined version of DeepPartial.